cmd/release-tools/release.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+script_name="${0##*/}"
+script_dir="$(dirname $(realpath -s "$0"))"
+release_tool="${script_dir}/release"
+owner=${OWNER:-clearcontainers}
+
+function usage() {
+cat << EOT
+Usage: ${script_name} <repo>
+
+Enviroment Variables:
+
+TOKEN: Export GITHUB_TOKEN variable with a valid token and repository permissions
```

Comment:
maybe we could verify if GITHUB_TOKEN and GOPATH var are defined

Reply:
This is only mandatory for ./release-tool release <repo>. Check https://github.com/clearcontainers/tests/pull/778#discussion_r155662528 :).